sci-calculators/bc-gh/bc-gh-6.2.2.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="
+	https://git.yzena.com/gavin/bc/
+	https://github.com/gavinhoward/bc/
+"
+SRC_URI="
+	https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz
+"
+S="${WORKDIR}/bc-${PV}"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="libedit readline"
+
+DEPEND="
+	!readline? (
+		libedit? ( dev-libs/libedit:= )
+	)
+	readline? (
+		sys-libs/readline:=
+		sys-libs/ncurses:=
+	)
+"
+RDEPEND="
+	${DEPEND}
+"
+
+src_configure() {
+	local myconf=()
+	if use readline ; then
+		myconf+=( -r )
+	elif use libedit ; then
+		myconf+=( -e )
+	fi
+
+	local -x EXECSUFFIX="-gh"
+	local -x PREFIX="${EPREFIX}/usr"
+	./configure.sh -pGNU "${myconf[@]}" -GTlP || die
```

Comment:
Could you move all configure options into `myconf`, and document them with comments? The short options definitely aren't self-evident, i.e. something like:

```
local myconf=(
	# does something GNU-related
	-pGNU
	# does something else
	-G
	# blah blah
	-T
)
```
etc. (leaving `-r`/`-e` as-is is fine since `use readline`/`use libedit` makes their purpose clear)